### PR TITLE
 Select from events based upon their displayLevel value

### PR DESCRIPTION
**Pull request by lpil**
candles

- [x] Remove ticket container
- [x] Pull featuredEvent from events by looking for an event with the `"Featured"` display level

## Notes

The site will look the same and display the same information as before. From the user's point of view nothing has changed.

## Test plan

- Remove the `featuredEvent` from React London. The site will still render the same as this property is no longer used.
- Set all `displayLevel` properties on RL events to be not `Featured`. The site will render successfully without information as no event has been picked to display.
- Set one RL event `displayLevel` to `Featured`. This one will be displayed by the site.

### CODE REVIEW

File: shared/utilities/ticket-status/index.js [hunk 1]
```diff
@@ -1,6 +1,37 @@
 import moment from 'moment';
 import { isBefore, isAfter, formatDate } from '../date';
 
+const defaultOptions = [
+  {
+    title: '',
+    ticketsAvailable: false,
+    waitingListOpen: false,
+    displayLevel: 'Featured',
```
Comment (pixelblend) [inline shared/utilities/ticket-status/index.js]: Would this accidentally put half-written event information into the featured list?

Comment (lpil) [inline shared/utilities/ticket-status/index.js]: This data is used to select from the states below (see the one linked below), so you always get a valid message (though possibly not the right one if data is missing).

https://github.com/redbadger/react.london/blob/5968bddca1ad9b1cb09b3fd3b9b6bed5d2be350f/shared/utilities/ticket-status/index.js#L61

File: server/data/badger-brain.js [hunk 2]
```diff
@@ -20,56 +20,57 @@ export const siteStateQuery = `
...
-      }
-      endDateTime {
-        iso
-      }
-      ticketReleaseDate {
-        iso
-      }
-      externalLinks {
-        url
+      summary
```
Comment (AndrewBestbier) [inline server/data/badger-brain.js]: weird indentation?

Comment (lpil) [inline server/data/badger-brain.js]: Thanks!

**Comment by lpil** [comment]
Good to merge. Just waiting for the ticket release to happen before merging in.

**Pull request merged**